store/src/java/com/zimbra/qa/unittest/TestZClient.java

```diff
@@ -29,10 +33,10 @@
 import java.util.List;
 import java.util.Set;
 
-import junit.framework.Assert;
-import junit.framework.TestCase;
-
 import org.apache.commons.io.IOUtils;
+import org.junit.After;
```

Comment:
Changes now much smaller.  Could do with JUnit 4 changes at some point - but probably better on the develop branch.